docs/wymeditor_development/making_releases.rst

```diff
@@ -155,9 +153,14 @@ Build WYMeditor
 #. Check that examples work when served from ``dist/`` by using ``grunt
    server:dist``.
 
-#. Commit the changes in ``dist/``.
```

Comment:
This is incomplete. Source changes need to be committed as well.

Replaced this with a whole section, below.